### ISSUE TITLE
Keyboard navigation cannot reach focusable html elements added via useHTML:true

### ISSUE THREAD

**Issue by MikeGodin** [highcharts/highcharts#18471]
#### Expected behaviour
Users can use keyboard navigation to reach focusable html elements added via `useHTML:true`

#### Actual behaviour
Keyboard navigation skips over focusable html elements added via `useHTML:true`

#### Live demo with steps to reproduce
https://jsfiddle.net/MikeGodin/x0ke8scn/

Click on button "Click here first, then tab" then use keyboard navigation to attempt to reach links in title and along x-axis.

#### Product version
HighCharts v10.3.3 (2023-01-20), using Accessibility module.

#### Affected browser(s)
All

**Comment by hihajdus** [comment]
Hi [MikeGodin](https://github.com/MikeGodin), 
Thanks for report

**Comment by hihajdus** [comment]
Hi @MikeGodin, 
Thanks for reporting this.

_These elements are not interactive, so they do not need tab focus and through our user testing (again, especially for blind users), it is confusing if we make them available. It is better to provide the same information in a structured way before the chart._

More you can read at this [topic](https://www.highcharts.com/forum/viewtopic.php?t=43887#p155254) from forum and our [blog](https://www.highcharts.com/blog/accessibility/) article describing accessibility features.
